src/_pytest/pytester.py

```diff
@@ -547,8 +547,10 @@ def __init__(
 
     def __repr__(self) -> str:
         return (
-            "<RunResult ret=%s len(stdout.lines)=%d len(stderr.lines)=%d duration=%.2fs>"
-            % (self.ret, len(self.stdout.lines), len(self.stderr.lines), self.duration)
+            f"<RunResult ret={self.ret!s} "
```

Comment:
Is the `!s` needed? I assume it is the default, so we can remove it?

Reply:
It can show an enum so it's different with `!s` (``1`` vs ``<EnumName.ValueName>``  (detected due to a test that also has ``!s``)